elf/perf.go

```diff
@@ -163,6 +163,11 @@ func (pm *PerfMap) PollStart() {
 		pageSize := os.Getpagesize()
 		state := C.struct_read_state{}
 
+		defer func() {
+			close(pm.receiverChan)
+			close(pm.lostChan)
```

Comment:
Right. It works for tcptracer-bpf but I forgot about the generic case. I'll fix it.